README.md

```diff
@@ -436,6 +528,331 @@ Allows detailed CloudWatch monitoring to be enabled for the current instance.
 
     aws_instance_monitoring "enable detailed monitoring"
 
+
+## aws_cfn_stack
```

Comment:
can we call this aws_cloudformation_stack even though it's a bit more to type. It's much more clear when the user reads over cookbook code.